### DISCUSSION TITLE
Button Animations

### DISCUSSION THREAD

**Discussion by darktempla** [huntabyte/shadcn-svelte#494] [Help] [answered]
First and foremost I would like to commend the authors on a good job on these components. Consistent, simple but classy. Doing research everyone usually bangs on about Skelton etc. Not taking anything away from the other frameworks but the themes most of them provide you out of the box are pretty unusable. Out of the box I feel these components can be dropped in with very little mods. 

The only issue I have been able to fault from a UX point of view is the lack of subtle animations in particular with the button. 

It would be nice to have a visible queue to the user that the button has been clicked. Most other frameworks would add a slight shrink/grow animation etc for example. 

New to this framework so hoping someone could give me some guidance on the best way to approach this. Any help would be appreciated.

**Comment by huntabyte** [comment, marked as answer]
Hey there!

We won't add them to the library at this time, but you're more than welcome to edit your components to do so!

You could add `active:scale-95` or similar to the button classes to accomplish that!

**Reply by darktempla** [reply]
Cool thanks for the reply got it working as suggested.
